src/pymovements/utils/parsing.py

```diff
@@ -349,19 +352,20 @@ def parse_eyelink(
                     # each metadata pattern should only match once
                     compiled_metadata_patterns.remove(pattern_dict)
 
-    if not metadata:
-        raise Warning('No metadata found. Please check the file for errors.')
-
     # if the sampling rate is not found, we cannot calculate the data loss
     actual_number_of_samples = len(samples['time'])
+    # if we don't have any recording config, we cannot calculate the data loss
+    sampling_rate = _check_sampling_rate(recording_config)
```

Comment:
please assign this to `metadata['sampling_rate']` for backwards-compatibility

Reply:
I changed my mind, backwards compatibility isn't really needed for that field as its now passed to `Experiment` in `from_asc()`.